src/BasicLogger.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace DigitalRisks\LaravelEventStore;
+
+use DigitalRisks\LaravelEventStore\Contracts\Logger;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Support\Facades\Log;
+
+class BasicLogger implements Logger
+{
+    public function eventStart($eventOrType, $payloadOrEvent)
+    {
+        $url = parse_url(config('eventstore.http_url'));
+        $url = "{$url['scheme']}://{$url['host']}:{$url['port']}/web/index.html#";
+
+        $type = $eventOrType;
+        $event = $payloadOrEvent;
+
+        if (is_object($eventOrType)) {
+            $type = get_class($eventOrType);
+            $event = $eventOrType->getEventRecord();
+        }
+
+        $stream = $event->getStreamId();
+        $number = $event->getNumber();
+        $hasListeners = Event::hasListeners($type);
+
+        Log::info("{$url}/streams/{$stream}/{$number}", ['type' => $type, 'hasListeners' => $hasListeners]);
```

Comment:
No need to pull in Log class, you can use `info("{$url}/streams/{$stream}/{$number}", ['type' => $type, 'hasListeners' => $hasListeners]);`

Reply:
would rather be more verbose and Taylor will probably remove that one day haha

Reply:
Fair Does, I agree with @eithed comment on it being in a Service Provider. Maybe see https://github.com/digitalrisks/laravel-eventstore/pull/18

Reply:
Yeah will move over to be a callback